lib/test.go

```diff
@@ -291,3 +291,23 @@ func ReceiveBallot(t *testing.T, nodeRunner *NodeRunner, ballot *Ballot) error {
 	err = nodeRunner.handleBallotMessage(ballotMessage)
 	return err
 }
+
+type TestBroadcastor struct {
+	Messages map[string]common.Message
+	recv     chan struct{}
+}
+
+func NewTestBroadcastor(r chan struct{}) TestBroadcastor {
+	p := TestBroadcastor{}
+	p.Messages = make(map[string]common.Message)
+	p.recv = r
+	return p
+}
+
+func (b TestBroadcastor) Broadcast(message common.Message) (errs map[string]error) {
+	b.Messages[message.GetHash()] = message
+	if b.recv != nil {
+		b.recv <- struct{}{}
+	}
+	return
+}
```

Comment:
So... That broadcast has no effect anymore ?

Reply:
@Geod24
Which kind of effect did you mean?
FYI, The Broadcaster is used in [TestStateINITTimeoutNotProposer](https://github.com/bosnet/sebak/pull/340/files#diff-cfb80960eeeaa7a01e196aea83c70705R83).